bus-mapping/src/evm/opcodes/return_revert.rs

```diff
@@ -189,31 +192,22 @@ fn handle_copy(
 
     let rw_counter_start = state.block_ctx.rwc;
 
-    let (src_begin_slot, src_full_length, src_shift) =
-        Memory::align_range(source.offset, copy_length);
-    let (dst_begin_slot, dst_full_length, dst_shift) =
-        Memory::align_range(destination.offset, copy_length);
-
-    let full_length = max(src_full_length, dst_full_length);
+    let mut src_range = MemoryWordRange::align_range(source.offset, copy_length);
+    let mut dst_range = MemoryWordRange::align_range(destination.offset, copy_length);
+    src_range.ensure_equal_length(&mut dst_range);
```

Comment:
`src_range`  and `dst_range` both use the same `copy_length`, the length is always equal . don't need to check `ensure_equal_length`

Reply:
that's not true, the offset is different, hence it may has different length after align

Reply:
I see!